modules/social_features/social_event/modules/social_event_managers/social_event_managers.module

```diff
@@ -186,17 +186,19 @@ function social_event_managers_menu_local_tasks_alter(&$data, $route_name) {
  * Implements hook_form_alter().
  */
 function social_event_managers_form_node_event_form_alter(&$form, FormStateInterface $form_state, $form_id) {
-  // Set author of event as event organizer automatically.
+  // Set author of event as event manager automatically.
   $config = \Drupal::configFactory()
     ->getEditable('social_event_managers.settings');
 
   if ($config->get('author_as_manager')) {
-    $account = \Drupal::currentUser();
-    $user = \Drupal::entityTypeManager()
-      ->getStorage('user')
-      ->load($account->id());
-
-    $last_key = $form['field_event_managers']['widget']['#max_delta'];
-    $form['field_event_managers']['widget'][$last_key]['target_id']['#default_value'] = $user;
+    if ($form_state->getTriggeringElement() === NULL) {
```

Comment:
This is needed because if you click "Add more" it would also fill in the author again. Even if you already added yourself as an event manager.